superset-frontend/src/components/Button/index.tsx

```diff
@@ -144,6 +144,17 @@ export default function Button(props: ButtonProps) {
     colorHover = primary.base;
   }
 
+  const element = children as ReactElement;
+
+  let renderedChildren = [];
+  if (element && element.type === React.Fragment) {
+    renderedChildren = Children.toArray(element.props.children);
+  } else {
+    renderedChildren = Children.toArray(children);
+  }
+
```

Comment:
```suggestion
const renderedChildren = Children.toArray(element?.type == React.Fragment ? element.props.children : children);
```
Not sure if it's easier to read in a one liner and/or with a null-coalescing operator. ¯\\\_(ツ)_/¯